smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/action/WriteFileAction.java

```diff
@@ -73,7 +87,11 @@ protected void execute() throws Exception {
         String.format(
             "Action starts at %s : Write %s with length %s",
             Utils.getFormatedCurrentTime(), filePath, length));
-    final OutputStream out = dfsClient.create(filePath, true);
+
+    Path path = new Path(filePath);
+    FileSystem fileSystem = path.getFileSystem(conf);
+
+    final OutputStream out = fileSystem.create(path, true);
```

Comment:
Replication may be default (3). Please refine this code according to [CopyFileAction](https://github.com/Intel-bigdata/SSM/blob/trunk/smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/action/CopyFileAction.java#L188)